chips/sam4l/src/gpio.rs

```diff
@@ -79,6 +80,8 @@ pub enum PeripheralFunction {
 const BASE_ADDRESS: usize = 0x400E1000;
 const SIZE: usize = 0x200;
 
+pub static INTERRUPT_COUNT: AtomicUsize = AtomicUsize::new(0);
```

Comment:
Why is this a `usize` instead of a `bool`? Interrupts can only ever be enabled or disabled.